website/cue/reference/components/sources/opentelemetry.cue

```diff
@@ -98,6 +98,34 @@ components: sources: opentelemetry: {
 		}
 	}
 
+	configuration_examples: [
+		{
+			title: "OTLP Defaults"
+			configuration: {
+				opentelemetry: {
+					grpc: {
+						address: "0.0.0.0:4317"
+						tls: {
+							enabled:  true
+							ca_file:  "/var/vector/ca.pem"
```

Comment:
The `ca_file` shouldn't be needed unless they want to do client certificate verification (not super common in my experience).

Reply:
Ah, gotcha, if it is common in OTel land then makes sense to include. I'd suggest `/etc/ssl/certs/vector_ca.pem`

Reply:
Also just of the thoughts of "more example = better"